README.md

```diff
@@ -201,12 +201,18 @@ following non-zero codes otherwise:
 |---------|-------|
 |1|Default failure code. An unrecoverable error was encountered.|
 |2|Phylum analysis is complete and contains a policy violation.|
+|5|Phylum analysis is incomplete. Only used when enabled [by option][script_options].|
 |6|Phylum analysis is incomplete and contains a policy violation.|
 |10|Dependency file(s) failed filtering and excluded from analysis. See [this FAQ][FAQ] for more.|
 |11|No dependency files were provided or detected.|
 |12|No dependencies found in any current dependency file.|
 |20|A manifest is attempted to be parsed but lockfile generation has been disabled.|
 
+Exit codes of 10 or higher represent situations not directly linked to Phylum analysis. These errors are important
+because they indicate a complete Phylum analysis was not possible, which is cause for further investigation and
```

Comment:
I don't like the word "cause" here, because it kinda sounds like someone else is going to do it.

I think `which necessitates further investigation.` is the better phrasing? The "correction" is a bit odd too since it feels like there's something missing like "error correction".

Reply:
It has been changed.